communitiesuk/funding-service-design-frontend
# Fs-3392 feedback welsh translations

Fs-3392 feedback welsh translations

### Change description
- Added welsh translations to all welsh survey feedback forms
- Critical Label bugs should be fixed

### How to test
- open a COF application in welsh
- complete a section which has required feedback
- open feedback form and all should be translated in welsh
- complete entire application and open final survey feedback at the bottom
- End of application Feedback form should be in  welsh
